site2/docs/concepts-messaging.md

```diff
@@ -280,6 +292,77 @@ message = consumer.receive();
 consumer.acknowledge(message);
 ```
 
+### Retry letter topic
+
+Retry letter topic allows you to re-consuming message even some messages are not consumed successfully. The messages that are failed to be consumed are stored in a specific topic, you can custom retry delay time for the message to be retried, and subscribers will automatically subscribe to the retry letter topic and re-consume it, when the specified maximum number of retries is reached, the message will send to a dead letter topic for manual processing.
```

Comment:
```suggestion
Retry letter topic allows you to store the messages that failed to be consumed and retry consuming them later. With this method, you can custom the interval to retry the delivery of the messages, and subscribers automatically subscribe to the retry letter topic and re-consume it. When the specified maximum number of retries is reached, the messages are moved to a dead letter topic for manual processing.
```

Reply:
Suggest we add an illustration to demonstrate the flow of the retry letter topic and describe its difference with the delayed message queue.

```
The diagram below illustrates the concept of the retry letter topic.
[Placeholder of Image]

The intention of using retry letter topic is different from using [delayed message delivery](#delayed-message-delivery), even though both are aiming to consume a message later. Retry letter topic serves failure handling through message redelivery to ensure critical data is not lost, while delayed message delivery is intended to deliver a message with a specified time of delay.
```